sass/vapor/_MultiSelect.scss

```diff
@@ -1,7 +1,55 @@
-@import "../bourbon/bourbon";
-@import "../../node_modules/@material/select/dist/mdc.select.min.css";
+@import '../coveo-styleguide/scss/common/palette';
+@import '../bourbon/bourbon';
+@import '../Variables';
 
-.mdc-list-item {
+.coveo-multi-select {
+  border: $default-low-contrast-border;
+  width: 250px;
+  padding: 0;
+  outline: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+
+  .coveo-list-group {
+    color: $color-light-contrast-grey;
+    margin: 16px 0 0;
+    padding: 0 0 0 16px;
+    font-weight: 400;
+
+    &:last-child {
+      margin-bottom: 16px;
+    }
+  }
+
+  .coveo-list-item {
+    color: $dark-grey;
+    margin: 0 0 0 -16px;
+    padding: 0 16px;
+
+    &:checked {
+      background-color: transparent;
+    }
+
+    &:first-child {
+      margin-top: 12px;
+    }
+
+    &:last-child {
+      margin-bottom: 8px;
+    }
+  }
+
+  &:focus {
+    .coveo-list-item {
+      &:checked {
+        background-color: $coveo-blue;
+      }
+    }
+  }
+}
```

Comment:
All of the above CSS was taken directly from `@material/select` and renamed. It could be improved, but I figured this might be out of scope for the purpose of this PR. If you prefer, I could improve the style of this component.